requirements/pip.txt

```diff
@@ -8,7 +8,7 @@ wheel==0.37.1
     # via -r requirements/pip.in
 
 # The following packages are considered to be unsafe in a requirements file:
-pip==21.3.1
+pip==22.0.2
```

Comment:
No major deprecation in the new version https://pip.pypa.io/en/stable/news/#deprecations-and-removals. Seems safe to be upgraded.